streamparse/cli/submit.py

```diff
@@ -171,11 +171,12 @@ def submit_topology(name=None, env_name=None, options=None, force=False,
 
     # If using virtualenv, set it up, and make sure paths are correct in specs
     if use_venv:
+        virtualenv_override_name = env_config.get('virtualenv_name', override_name)
```

Comment:
Nitpick: I'd rather we just stick with `virtualenv_name` like the CLI option is called.